### PR TITLE
fix(errorCode): add error code for cloud service error

### PR DESCRIPTION
**Pull request by PeachThinking**
#### What type of PR is this?
type-bug

#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by yhilmare — APPROVED** [review]
lgtm

File: server/odc-core/src/main/resources/i18n/ErrorMessages.properties [hunk 1]
```diff
@@ -67,6 +67,7 @@ com.oceanbase.odc.ErrorCodes.ConnectionReadonly=Access denied. The current data
 com.oceanbase.odc.ErrorCodes.ConnectionTempOnly=Access denied. Creating persistent data sources is not allowed.
 com.oceanbase.odc.ErrorCodes.ConnectionFlowConfigNotExists=No approval process found for the current data source, please contact the administrator.
 com.oceanbase.odc.ErrorCodes.ConnectionUnsupportedDBVersion=Unsupported database version: {0}.
+com.oceanbase.odc.ErrorCodes.TenantConnectionAddressNotFind=The tenant has no connectable address. Please go to the OBCloud Console to create a private/public network connection.
```
Comment (JessieWuJiexi) [inline server/odc-core/src/main/resources/i18n/ErrorMessages.properties:70]: There are no available endpoints for this tenant. Please navigate to the OceanBase Cloud console to create a private or public endpoint.

**Review by yizhouxw — DISMISSED** [review]
lgtm

File: server/odc-core/src/main/resources/i18n/ErrorMessages.properties (hunk 1 above)
Comment (PeachThinking) [reply server/odc-core/src/main/resources/i18n/ErrorMessages.properties]: ok～

**Review by JessieWuJiexi — APPROVED** [review]

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**